sidebars.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
+const sidebars = {
+  docs: [    {
+      type: 'autogenerated',
+      dirName: '.', // '.' means the current docs folder
+    },
+    {
+      type: 'category',
+      label: 'Community',
+      items: [
+        {
+          type: 'link',
+          label: 'Stencil on Twitter',
+          href: 'https://twitter.com/stenciljs',
+        },
+        {
+          type: 'link',
+          label: 'Stencil on Slack',
+          href: 'https://stencil-worldwide.herokuapp.com/',
+        },
+        {
+          type: 'link',
+          label: 'Stencil on GitHub',
+          href: 'https://github.com/ionic-team/stencil',
+        },
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Legal',
+      items: [
+        { 
+          type: 'link',
+          label: 'Telemetry',
+          href: "/telemetry"
+        },
```

Comment:
in order to get a top-level link (`/telemetry`) and not have to live at `/docs/telemetry` this page had to be moved out of the `docs/` directory and into `src/pages/` and a link manually added like this.